pkg/nvlib/info/builder.go

```diff
@@ -29,11 +37,21 @@ func New(opts ...Option) Interface {
 	if b.root == "" {
 		b.root = "/"
 	}
+	if b.nvmllib == nil {
+		b.nvmllib = nvml.New(
+			nvml.WithLibraryPath(b.root.tryResolveLibrary("libnvidia-ml.so.1")),
```

Comment:
should we have a constant for this? Does it actually make sense to define the constant in go-nvml itself?

Reply:
We have:
```
	defaultNvmlLibraryName      = "libnvidia-ml.so.1"
```
in go-nvml. I suppose we can make it public.